src/Domain/Stager.php

```diff
@@ -84,12 +108,28 @@ private function validatePreconditions(): void
         }
     }
 
-    private function runCommand(): void
+    /**
+     * @throws \Symfony\Component\Process\Exception\RuntimeException
+     */
+    private function runCommand(): string
     {
-        $this->processFactory
+        $process = $this->processFactory
             ->create(array_merge([
                 'composer',
```

Comment:
Does this not assume that Composer is installed globally? I'd imagine it would be useful to specify the path, in some cases, to a specific Composer binary.

Reply:
> or I could require composer/composer and call its bin script. I'm not fond of adding composer/composer to the production requirements

I definitely do not think we should require Composer as a production dependency (almost certainly as a dev dependency). IMHO, we should use the ExecutableFinder _and_ allow power users to specify a path to a binary.

Reply:
I've refactored to use `ExecutableFinder`. Power users can override the binary used by simply adding it to the `$PATH` in their shell environment.